docs/BOOT_LOADER_INTERFACE.md

```diff
@@ -58,6 +58,17 @@ variables. All EFI variables use the vendor UUID
 * The EFI variable `LoaderEntryDefault` contains the default boot loader entry
   to use. It contains a NUL-terminated boot loader entry identifier.
 
+* The EFI variable `LoaderEntrySysFail` contains the sysfail boot loader entry
```

Comment:
`the sysfail boot loader entry` sounds ambiguous, as `sysfail` is not declared at this point...